static/src/javascripts/projects/common/modules/commercial/dfp-api.js

```diff
@@ -290,8 +292,12 @@ define([
             googletag.enableServices();
             // as this is an single request call, only need to make a single display call (to the first ad
             // slot)
-            googletag.display(keys(slots).shift());
-            displayed = true;
+            if (prebidService.testEnabled) {
```

Comment:
Shouldn't this rather be `if (prebidService.testEnabled && prebidService.slotIsInTest(slotKey))` to avoid `loadSlot` to fail when trying to access a non-existing `slots` array?